### ISSUE TITLE
Change back the getting started column

### ISSUE THREAD

**Issue by Sylvhem** [mastodon/mastodon#7825]
[Mastodon 2.4.1](https://github.com/tootsuite/mastodon/releases/tag/v2.4.1) introduced some major changes to the getting started column in #7676. If there is some good ideas, my personal opinion on that matter is that those changes made the menu a lot less useful.

1. The getting started menu now uses the same icon as the lists and the list menu. This is confusing.
2. The lists of muted users, blocked users and hidden instances are not accessible through this menu anymore. Even worse, they are now hidden in a totally unrelated menu, that is only accessible by opening your profile, i.e., clicking on your own handle somewhere on the interface. On top of that, there is absolutely no clue anywhere om where to find those options. Since Mastodon tries to fight harassment and abuse, I think it's a necessity that those options are located in obvious place.
3. A lot of people used to complain that, since the first redesign of the landing page, there was no obvious link to /about/more (which usually contains the instance's rules) on the front page anymore. The only obvious link was in the getting started menu and thus, could only be seen *after* the initial registration. With this update, there isn't even a prominent link on getting started anymore. Instead, the link have been relegated to the column footer where it is lost among all the other links.
4. Like the "About this instance" entry, the logout button is now lost on the column *footer*, something that seems really strange and out of place to me. I really don't think the link to logout should be in Mastodon's footer. Moreover, placing it here creates a discrepancy in which elements are displayed in the quick access menu. Until now, all the elements of this menu where elements of the getting started's menu and that's not really the case anymore.
5. The "Preferences" entry is in a similar situation than the logout one, except it's not even on the footer (that's probably better, but it's even less visible).
6. Entries corresponding to the currently pinned columns used to be removed from the getting started menu. That's not the case anymore.

Since those changes were introduced to make room for the now discarded trending functionality, I would suggest to revert back those change altogether.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by ariasuni** [comment]
Other things that’s bothering me:

## Icons

- The getting started button, which open a pane, and the «menu» button in the, which opens a menu, use the same icon despite different behavior.
- Also, it highlights that it’s weird to have the icon of a navigation button smaller than the icon of a toolbar button.

## Where elements should be

- Lots of things are relegated to the menu on the user’s profile, but there’s no obvious way to get there, and it might be difficult to find. I think that everything that’s opening a new pane could be in the getting started pane, since it’s so empty now.
- The user menu could still contains elements closely related to the actual view, like «Edit profile», «Pinned toots» and «Favourites».

## Footer

Documentation and help is important for onboarding, but keyboard shortcuts for example are much less visible now. We may want to think carefully about what’s displayed or not in the footer to improve discoverability.

**Comment by Gargron** [comment]
Have you seen this?

![grafik](https://user-images.githubusercontent.com/184731/41678353-27c5bc18-74cb-11e8-9b10-2182eefbe131.png)

I think the icon for "getting started" is indeed a contested topic. Initially it was an asterisk, because I thought it's like a footnote for the page. But on mobile, the same page is used as an extended menu, and some people also said that it's weird that it's an asterisk instead of a hamburger icon, so I changed it. Now the icon for "lists" has been changed slightly so they're separate, everything sorta makes sense again.

Another thing is that having such a large menu is wasteful. The buttons are huge. Putting every bit of navigation in such a wasteful format is a bad idea, so I decided to highlight the most important - discovery - items. Furthermore, there is an accepted design idiom with the footer text links, you'll find on every other big site, so it made sense to use that idiom to include more links that were previously not there at all.

I also like the fact that getting started is no longer the same height as other columns. I believe it puts an emphasis on the other 3, so the visitor feels like there's 3 columns and not 4, which is less overwhelming - if I may put it this way, the height difference highlights the fact that the getting started column is a bonus.
